curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e5eea8261d22856ead1c.md

```diff
@@ -255,8 +255,7 @@ function calculateCalories(e) {
 
   const consumedCalories = breakfastCalories + lunchCalories + dinnerCalories + snacksCalories;
   const remainingCalories = budgetCalories - consumedCalories + exerciseCalories;
-  const surplusOrDeficit = remainingCalories >= 0 ? 'Surplus' : 'Deficit';
-
+	const surplusOrDeficit = remainingCalories < 0 ? 'Surplus' : 'Deficit';
```

Comment:
not sure if this indentation was caused by me or the linter, please let me know if its an issue and i can go back fix them

Reply:
we should look into this indentation difference because it shouldn't be here

Reply:
I am assuming you did a search and replace on only the english files. 
So you might have added additional spacing on accident during that search and replace.